src/AspNet.Security.OAuth.WeChat/WeChatAuthenticationInitializer.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using JetBrains.Annotations;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
+
+namespace AspNet.Security.OAuth.WeChat
+{
+    public class WeChatAuthenticationInitializer : IPostConfigureOptions<WeChatAuthenticationOptions>
+    {
+        private IMemoryCache _cache;
+
+        /// <summary>
+        /// Creates a new instance of the <see cref="WeChatAuthenticationInitializer"/> class.
+        /// </summary>
+        public WeChatAuthenticationInitializer([NotNull] IMemoryCache cache)
+        {
+            _cache = cache;
+        }
+
+        /// <summary>
+        /// A ugly hack to deal with WeChat MP OAuth state length limitation.
```

Comment:
This is an internal implementation detail, so shouldn't be in public-facing XML Intellisense comments.